File: GFG-fibonacci/fibonacci.c
#include<stdio.h>

int main()
{
    int a, b, n, c, s;
    a = -1;
    b = 1;
    s = 0;
    printf("enter n: ");
    scanf("%d",&n);
    
    while (s!=n)
    {
        c = a+b;
        printf("%d\n",c);
        a = b;
        b = c;
        s++;
    }
    return 0;
}

File: GFG-fibonacci/fibonacci.java
import java.util.Scanner;
class Fibonacci
{	
	public static void main(String args[])
	{
		int a, b, n, c, s;

        System.out.print("enter n : ");
        
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        
        a = -1;
        b = 1;
        s = 0;
        
        while (s!=n)
        {
            c = a+b;
            System.out.println(c);
            a = b;
            b = c;
            s++;
        }
	}
}